chart/amplication/templates/_helpers.tpl

```diff
@@ -0,0 +1,177 @@
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "amplication.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "amplication.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+
+{{/* 
+K8s-wait-for init container functions 
+*/}}
+
+# Waiting for the postgres connection to become available 
+{{- define "waitfor.postgres.connection" }}
+  # If postgres server is installed within cluster we'll wait for pods to be ready.
+  {{ if eq .Values.postgres.install true  }}
+- name: wait-for-postgres-connection
+  image: groundnuty/k8s-wait-for:v1.3
+  imagePullPolicy: Always
+  args:
+    - "pod"
+    - "{{ .Values.postgres.fullnameOverride }}-0"
+  {{ end }}
+{{- end }}
+
+
+# #########################
+# #### Amplication App ####
+###########################
+
+{{/*
+Expand the name of the Amplication app service.
+*/}}
+{{- define "amplication.app.name" -}}
+{{ include "amplication.fullname" . }}-{{ .Values.app.name }}
+{{- end }}
+
+{{/*
+Amplication App Common labels
+*/}}
+{{- define "amplication.app.labels" -}}
+helm.sh/chart: {{ include "amplication.chart" . }}
+{{ include "amplication.app.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Amplication App Selector labels
+*/}}
+{{- define "amplication.app.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "amplication.app.name" . }}
+{{- end }}
+
+
+# #########################
+# #### Amplication Scheduler ####
+###########################
+
+{{/*
+Expand the name of the Amplication app service.
+*/}}
+{{- define "amplication.scheduler.name" -}}
+{{ include "amplication.fullname" . }}-{{ .Values.scheduler.name }}
+{{- end }}
+
+{{/*
+Amplication App Common labels
+*/}}
+{{- define "amplication.scheduler.labels" -}}
+helm.sh/chart: {{ include "amplication.chart" . }}
+{{ include "amplication.scheduler.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Amplication App Selector labels
+*/}}
+{{- define "amplication.scheduler.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "amplication.scheduler.name" . }}
+{{- end }}
+
+
+# ##############################################
+# #### Amplication Docker Container Builder ####
+################################################
```

Comment:
This probably should not be part of the chart -  the container builder is the component that builds the container of the generated app (and it is used internally as a package in the Amplicaiton server)
We can discuss separately to consider adding support within the package to generate the required configuration and settings for helm for the generated container

Reply:
Would it be helpful to have an optional install flag? Something like `containerbuilder.install: false/true`. My only issue with not including some sort of builder is when I started the amplication server, by default it was expecting a `/var/run/docker.sock` coming from the `dockerode` package. Basically preventing me from signing up and using the app.

Reply:
OK. 
I will follow the instructions to run on my machine to better understand the entire process and will follow up

Reply:
I now understand what you mean.
I thought "container-builder" is a container for "Amplication container builder" which is one of the packages in Amplication mono-repo. my confusion...

I approved the entire PR to a new branch on our repo to start playing with it. 
Looks great so far.

Few questions that I still trying to figure:
1. Is there a reason to use the chart for development? 
2. What is required in order to use the chart for our cloud deployment (currently on GCP, planning to move to AWS... it may be a great opportunity )
3. Are there any open items that still require attention?